hw/ip/edn/doc/theory_of_operation.md

```diff
@@ -66,7 +66,7 @@ For instance, if there are three boot-time request mode EDN's in the system, the
 Before entering auto request mode, it is the responsibility of firmware to first generate an `instantiate` command for the EDN-associated instance via the [`SW_CMD_REQ`](registers.md#sw_cmd_req) register.
 The required `generate` and `reseed` commands must also be custom generated by firmware and loaded into the respective command replay FIFOs via the [`GENERATE_CMD`](registers.md#generate_cmd) and [`RESEED_CMD`](registers.md#reseed_cmd) registers.
 These `generate` commands will be issued as necessary to meet the bandwidth requirements of the endpoints.
-The `reseed` commands will be issued once every `MAX_NUM_REQS_BETWEEN_RESEEDS` generate requests.
+The `reseed` commands will be issued once every `MAX_NUM_REQS_BETWEEN_RESEEDS` generate requests. The reset value is `0`. This implies that no generate commands will be issued in unless this value is changed by firmware.
```

Comment:
Please put one sentence per line only. It eases future diffs.

Reply:
will be issued in unless -> will be issued unless (remove "in")

Reply:
Thanks. This should be fixed now.